fem/fe.hpp

```diff
@@ -38,7 +38,8 @@ class BasisType
       OpenHalfUniform = 5,  ///< Nodes: x_i = (i+1/2)/n,   i=0,...,n-1
       Serendipity     = 6,  ///< Serendipity basis (squares / cubes)
       ClosedGL        = 7,  ///< Closed GaussLegendre
-      NumBasisTypes   = 8   /**< Keep track of maximum types to prevent
+      Integrated      = 8,  ///< Integrated indicator functions
```

Comment:
Yeah, I wasn't so happy with the name myself. Think of it as a placeholder for now. I'm not sure `Gerritsma` is any better, since it's not standard terminology, and you'd have to read the paper anyway to know what it means.

We could try to think of another term that suggests that instead of a *nodal* basis, it is a basis that satisfies a sort of "integral indicator function" condition. The other thing to note is that it is also dependent on the partitioning of the reference interval, which in this case is Gauss-Lobatto.

If we wanted to go for maximum verbosity, we could have `GaussLobattoIntegralIndicator`, which conveys the meaning but it awkward to read and write.

Reply:
I agree that we should have a slightly more descriptive name, e.g. `IntegratedGLL`, `IntegratedTensor`, or `IntegratedGerritsma`.

Reply:
I agree that we should have a slightly more descriptive name, e.g. `IntegratedGLL`, `IntegratedTensor`, or `IntegratedGerritsma`.

Reply:
My vote would be for `IntegratedGLL`.

Reply:
I changed it to `IntegratedGLL`. Let me know if that looks good to you.